bugbug/model.py

```diff
@@ -21,6 +21,7 @@
 from sklearn.feature_extraction.text import TfidfVectorizer
 from sklearn.metrics.classification import precision_recall_fscore_support
 from sklearn.model_selection import cross_validate, train_test_split
+from tabulate import tabulate
```

Comment:
You'll need to add this to the requirements.txt file (see the failing tests)

Reply:
added this, but still failing tests :(